json-schemas/workflowRunsAggregateRequest.json

```diff
@@ -0,0 +1,111 @@
+{
+    "type": "object",
+    "properties": {
+        "where": {
+            "type": "object",
+            "properties": {
+                "id": {
```

Comment:
I only have one field for workflow run ids. We will probably need to change this once we start querying more than one workflow for nextgen but that's a problem for later.

Reply:
1 field should still work right? Since each workflow run can only belong to 1 workflow anyways? Unless I'm wrong

Reply:
I think it just depends how we implement it from the frontend, e.g. make 4 agg queries (one for each workflow) or join all the workflow run ids together.